_sass/cylc/_header.scss

```diff
@@ -44,6 +44,10 @@
       a {
         @extend .flex;
         align-items: center;
+        &.active {
+          font-weight: bold;
+          text-decoration: underline;
```

Comment:
Not sure if the best way to display active item. We had "(Home)" as example before. I could try playing with CSS' `:before` and `:after` selector, then setting `content: "("` and `content: ")"`... I guess that would work in theory. This way was just much easier.

Reply:
underline is fine, definitely an improvement!